index.html

```diff
@@ -147,96 +173,112 @@ <h2>About</h2>
             aliquip ea eiusmod
           </p>
           <div class="about-butt">
-            <p>Get my CV</p>
+            <p class="about-butt-text">Get my CV</p>
             <img src="imgs/ic_download.png" alt="" />
           </div>
```

Comment:
- [ ] I would recommend replacing this ```p``` tag with a ```button``` or an ```anchor``` as the ```<a>``` tag defines a hyperlink, which is used to link from one page to another and also can be used to download embedded files on your webpage, 👍 👍